indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -174,6 +176,121 @@
 
 (def REINDEX_BATCH_SIZE 2000)
 
+(defn- suggestion-doc
+  [collection-id key-name value-map]
+  ; We want to pull Science Keywords from KMS because they are controlled
```

Comment:
For inline comments use 2 semicolons ;; and not one. https://wiki.earthdata.nasa.gov/display/CMR/Coding+Style+Guidelines.